python/tests/test_logistic_regression.py

```diff
@@ -305,9 +299,16 @@ def test_classifier(
     elasticNet_param: float = 0.0,
     tolerance: float = 0.001,
     convert_to_sparse: bool = False,
+    set_float32_inputs: Optional[bool] = None,
 ) -> LogisticRegression:
     standardization: bool = False
 
+    float32_inputs = set_float32_inputs
+    if float32_inputs is None:
+        float32_inputs = True
+        if data_type == np.float64 and max_record_batch > 1000:
```

Comment:
Add a comment explaining reason for this logic.

Reply:
Revised the use randomized value.